.gitignore

```diff
@@ -0,0 +1,36 @@
+.DS_Store
```

Comment:
Will this cause a problem if I have a windows computer??

Reply:
If you're using windows, you won't have any .DS_Store files! So it won't be a problem. The .gitignore tells git what files it should not track and send to github when you push. So none of the files listed to ignore are going to cause you any problems on windows!